sunpy/map/tests/test_plotting.py

```diff
@@ -153,6 +160,13 @@ def test_quadrangle_aia17_top_right(aia171_test_map):
     plt.legend()  # Check that the 'Quadrangle' label shows up in the legend
 
 
+@figure_test
+def test_quadrangle_aia17_pix_top_right(aia171_test_map):
+    aia171_test_map.plot()
+    aia171_test_map.draw_quadrangle(bottom_left=(50, 50)*u.pix,
+                                    top_right=(80, 90)*u.pix, edgecolor='cyan')
+
+
```

Comment:
```suggestion

@figure_test
def test_quadrangle_aia17_pix_top_right_different_axes(aia171_test_map):
    # Plot the map rotated by 30 degrees
    aia171_test_map.rotate(30*u.deg).plot()
    # Plot a rectangle in the pixel space of the original map
    aia171_test_map.draw_quadrangle(bottom_left=(50, 50)*u.pix,
                                    top_right=(80, 90)*u.pix, edgecolor='cyan')


```
And add the new figure hashes

Reply:
Keep the code comments (or something like them).  It's important so that some well-meaning developer doesn't misunderstand the point of the test and "fix" it in the future.

Reply:
Yes, will do :) thanks